### ISSUE TITLE
maximised Window movement between screens breaks cursor position

### ISSUE THREAD

**Issue by UltraBlackLinux** [WayfireWM/wayfire#1499]
**Describe the bug**
When moving a maximised window to a screen with a different resolution, the cursor position does not update for it, so you might end up clicking a different button, than what you are hovering over

**To Reproduce**
1. Open a window
2. Maximise it
3. Move it to a screen with a different resolution

**Expected behavior**
The cursor position should update

**Additional Information**
Opening and closing Expo corrects the cursor position

**Screenshots or stacktrace**
[Video](https://files.chonkyrabbit.eu/BugReports/wayfire-max-cursor.m4v)

**Wayfire version**
0.8.0-94f15048 (Apr 30 2022, branch 'master')

**Comment by ammen99** [comment]
Yes, I am quite sure there have been changes related to this, so unlikely to still be an issue. Feel free to reopen if you still experinece it on latest git.
